### PR TITLE
applications: Added support for groups in the Matter bridge

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
Added declaration of groups cluster and handling attribute read operations for the OnOff Light device.

### CODE REVIEW

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -198,7 +198,7 @@ Thingy:53: Matter weather station
 Matter Bridge
 -------------
 
-|no_changes_yet_note|
+* Added support for the groupcast communication to the On/Off Light device implementation.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]:
```suggestion
* Added support for groupcast communication to the On/Off Light device implementation.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -198,7 +198,7 @@ Thingy:53: Matter weather station
...
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:201]:
```suggestion
* Added support for groupcast communication to the On/Off Light device implementation.
```
Could also be "groupcast commands".

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Comment by kkasperczyk-no** [comment]
@wiba-nordic added one more change in the known issues section.

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -325,6 +325,16 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2
+
+KRKNWK-17360: The groupcast communication does not work for multiple endpoints being part of a group on a single Matter node
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:330]:
```suggestion
KRKNWK-17360: Groupcast communication does not work for multiple endpoints thare are part of a group on a single Matter node
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 4]
```diff
@@ -325,6 +325,16 @@ Matter
...
+  The logic in a Matter core implementation handles commands status in a wrong way for those targeted to a group.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:331]:
```suggestion
  The Matter core implementation handles commands status in a wrong way for those targeted to a group.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 5]
```diff
@@ -325,6 +325,16 @@ Matter
...
+  The issue is only visible in case of adding multiple endpoints, existing on the same Matter node, to the one group.
+  It results in application crash after receiving a group command.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:333]:
```suggestion
  This issue is only visible when adding multiple endpoints that exist on the same Matter node to the same group, and results in an application crash after receiving a group command.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 6]
```diff
@@ -325,6 +325,16 @@ Matter
...
+  In case of adding multiple Matter nodes with a single endpoint each to the group, the communication works correctly.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:334]:
```suggestion
  When adding multiple Matter nodes with a single endpoint each to the same group, the communication works correctly.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 7]
```diff
@@ -325,6 +325,16 @@ Matter
...
+
+  **Workaround:** Manually cherry-pick and apply commit from the main branch (commit hash: ``TODO: fill after merging sdk-connectedhomeip PR``).
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:336]:
```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``TODO: fill after merging sdk-connectedhomeip PR``).
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]

**Pull request merged**